docs/rfcs/014-delivery-channels-database.md

```diff
@@ -0,0 +1,200 @@
+
+In the above, the initial defaultDeliveryChannels uses two global/system policies and three policies that belong to the user, that they can edit. There are two that match on `image/*` but they are for different channels.
+
+An incoming image/tiff for customer 99 that didn't specify any deliveryChannels would be given:
+
+```
+  "deliveryChannels": [
+    {
+      "@type": "vocab:DeliveryChannel",
+      "channel": "iiif-img",
+      "policy": "default"
+    },
+    {
+      "@type": "vocab:DeliveryChannel",
+      "channel": "thumbs",
+      "policy": "https://api.dlcs.io/customers/99/deliveryChannelPolicies/thumbs/standard"
+    },
+  ]
+```
+
+There is also a one off DB-migration task, and that is to apply this new row creation in both tables to all EXISTING customers, so they get the data they would have if created with this mechanism already in place.
+
+These will act as a template and be copied (not referenced) for new customers when a new customer is created.
+
+## Creating new delivery channels
+
+customer creates custom av policy
+
+```
+POST /{base}/customers/20/deliveryChannelPolicies/iiif-av
+{
+   "name": "specific-mp4",
+   "displayName": "a specific policy for mp4",
+   "channel" : "iiif-av",
+   "policyData": "[\"video-mp4-1080p\"]",
+}
+```
+This produces a row in DeliveryChannelPolicies
+```
+     |--------- (constraint) ------------|
+| id | Customer | Channel | Name         | DisplayName               | System | PolicyCreated | PolicyModified | PolicyData            |
+|----|----------|---------|--------------|---------------------------|--------|---------------|----------------|-----------------------|
+| 1  | 20       | iiif-av | specific-mp4 | a specific policy for mp4 | 0      | (now)         | (now)          | [\"video-mp4-1080p\"] |
+```
+
+And there's now a DeliveryChannelPolicy resource at /customers/20/deliveryChannelPolicies/iiif-av/specific-mp4 that would look like this:
+```
+{
+    "@id": "https://api.dlcs.io/customers/20/deliveryChannelPolicies/iiif-av/specific-mp4",
+    "@type": "vocab:DeliveryChannelPolicy",
+    "name": "specific-mp4",
+    "displayName": "a specific policy for mp4",
+    "channel": "iiif-av",
+    "policyData": "[\"video-mp4-1080p\"]",
+    "policyCreated": "2024-01-24T15:36:58.6023600Z",
+    "policyModified": "2024-01-24T15:36:58.6023600Z"
+}
+```
+
+This can be used immediately for when you specify explicit delivery channels on an Asset:
+
+```
+{
+  "origin": "https://repository.org/films/my-movie.mov",
+  "deliveryChannels": [
+    {
+      "channel": "iiif-av",
+      "policy": "https://api.dlcs.io/customers/20/deliveryChannelPolicies/iiif-av/specific-mp4"
+    }
+  ],
+ 
+}  
+```
+But we like this policy so we want it to apply automatically on Space 5:
+
+```
+POST /{base}/customers/20/spaces/5/defaultDeliveryChannels
+{
+   "channel" : "iiif-av",
+   "policy": "https://api.dlcs.io/customers/20/deliveryChannelPolicies/iiif-av/specific-mp4",
+   "mediaType": "video/*"
+}
+```#

```

Comment:
Use the `View file` function to view rendered markup - something is breaking it. I think it's this trailing `#`